### ISSUE TITLE
Auto bug report from android crash

### ISSUE THREAD

**Issue by droberg** [nextcloud/android#5764]
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Can't toast on a thread that has not called Looper.prepare()
	at android.widget.Toast$TN.<init>(Toast.java:407)
	at android.widget.Toast.<init>(Toast.java:121)
	at android.widget.Toast.makeText(Toast.java:286)
	at android.widget.Toast.makeText(Toast.java:276)
	at android.widget.Toast.makeText(Toast.java:313)
	at com.owncloud.android.ui.activity.ConflictsResolveActivity.showErrorAndFinish(ConflictsResolveActivity.java:229)
	at com.owncloud.android.ui.activity.ConflictsResolveActivity.lambda$onStart$0$ConflictsResolveActivity(ConflictsResolveActivity.java:192)
	at com.owncloud.android.ui.activity.-$$Lambda$ConflictsResolveActivity$EM0Z1Vrhvmim7X4XXFnNqL4vK1A.run(Unknown Source:4)
	at java.lang.Thread.run(Thread.java:919)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30110090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Sony
Device: H8324
Model: H8324
Id: 52.1.A.0.618
Product: H8324

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 052001A000061802006556692

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Do you know what you did?
Seems to be a problem with conflict resolving dialog, but I am not sure how to reproduce it?

**Comment by droberg** [comment]
Sorry for the late answer. I don't use GitHub since a while back. 

I'm not sure exactly what I did. I remember having a real messy situation with several files in conflict. (After several crashed uploads it seems.. ). This was my first attempt at using the app, and I was trying to upload all my pictures/movies. The app got stuck in a unresponsive state a few times directly after enabling the directory sync and was killed of by Android OS. Then it turned out that it was actually doing something in the background, but extremely slowly (like some kB per second) so I let it run for a long-long-long time without touching it (and thus not triggering the unresponsive kill thingy.) After that it claimed to have synced everything except for a few large movies and some images (10Mb and about 250Mb).  During the attempts to solve this I tried several times but the same files kept coming back as in conflict, then it crashed.
